village/src/components/Smurfs.js

```diff
@@ -1,14 +1,33 @@
 import React, { Component } from 'react';
-
+import axios from 'axios';
 import Smurf from './Smurf';
 
 class Smurfs extends Component {
+  constructor() {
+    super();
+    this.state = {
+      smurfs: [],
+    };
+  }
+  componentDidMount = () => {
+    axios.get('http://localhost:3333/smurfs')
+    .then (response => {
```

Comment:
This should be on App.js too. Right now, you're pulling every smurf from the database every time you load the route containing this component. This defeats the purpose of having a single page application.